gptel-gemini.el

```diff
@@ -110,7 +110,10 @@
                   (list :text (string-trim
                                (buffer-substring-no-properties (point-min) (point-max)))))
             prompts))
-    (cl-callf (lambda (msg) (concat gptel--system-message "\n\n" msg))
+    (cl-callf (lambda (msg)
```

Comment:
Instead of this, you can avoid the whole `cl-callf` statement if no system message is available.